vampire_survivors/src/enemy.py

```
@@ -0,0 +1,19 @@
+        super().__init__(groups)
+        self.image = pygame.image.load('../assets/images/enemy.png').convert_alpha()
+        self.rect = self.image.get_rect(topleft = pos)
+
+        self.direction = pygame.math.Vector2()
+        self.speed = 3
+        self.player = player
+        self.health = 1
+
+        self.obstacle_sprites = obstacle_sprites
+
+    def update(self):
```

Ruff (0.12.2)

Missing return type annotation for public function `update`

Add return type annotation: `None`

(ANN201)